.cirrus.yml

```diff
@@ -32,9 +32,9 @@ env:
 # RE-USABLE CONFIGS
 #
 container_definition: &CONTAINER_DEFINITION
-  image: gcr.io/ci-cd-215716/base:latest
-  cluster_name: cirrus-uscentral1a-cluster
-  zone: us-central1-a
+  image: gcr.io/sonarqube-team/base:latest
+  cluster_name: cirrus-ci-cluster
```

Comment:
Your cluster name is to generic, should have use something like cirrus-ci-sq-cluster

Reply:
It is the `cirrus-ci-cluster` within the `sonarqube-team` project.